### PR TITLE
Feature/improve attribute builder

### PR DESCRIPTION
**Pull request by diba1013**

### CODE REVIEW

File: src/main/java/de/retest/recheck/file/FilePeer.java [hunk 1]
```diff
@@ -54,21 +52,27 @@ public FilePeer(final File file, ReChecker rechecker) {
...
 	@Override
 	public Set<Attribute> retrieveAttributes() {
-        try {
             return AttributeBuilder.create()
                     .add("size").value(FileUtils.sizeOf(file))
                     .add("readable").value(file.canRead())
                     .add("writable").value(file.canWrite())
                     .add("executable").value(file.canExecute())
-                    .add("crc32Checksum").valueHex(FileUtils.checksumCRC32(file))
+                    .add("crc32Checksum").tryValue(getHexChecksumCRC32())
```
Comment (roesslerj) [inline src/main/java/de/retest/recheck/file/FilePeer.java]: We need to be consistent: Either no method needs file as argument or all need it...

File: src/test/java/de/retest/helper/attribute/AttributeBuilderTest.java [hunk 2]
```diff
@@ -71,9 +70,58 @@ public void add_should_format_numbers_to_hex() throws Exception {
...
+            public void addValue(AttributeEntry callback) throws Exception {
+                throw new IllegalStateException("Thrown");
+            }
+        };
+        Set<Attribute> set = AttributeBuilder.create()
+                .add("exception").tryValue(function)
+                .build();
 
-        assertThat(set).contains(new Attribute("date", new SimpleDateFormat().format(date)));
+        assertThat(set).containsExactly(new Attribute("exception", "fallback"));
```
Comment (roesslerj) [inline src/test/java/de/retest/helper/attribute/AttributeBuilderTest.java]: Why is it a good idea to put an attribute with "exception"?

Comment (diba1013) [inline src/test/java/de/retest/helper/attribute/AttributeBuilderTest.java]: This is just a test. The important part is that the value is "fallback".

File: src/main/java/de/retest/recheck/file/FilePeer.java (hunk 1 above)
Comment (diba1013) [inline src/main/java/de/retest/recheck/file/FilePeer.java]: Do you mean that it should be `getHexChecksumCRC32(file)`

**Pull request merged**